binding/python/__init__.py

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright 2020 Picovoice Inc.
+#
+# You may not use this file except in compliance with the license. A copy of the license is located in the "LICENSE"
+# file accompanying this source.
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+from .rhino import Rhino
+from .util import *
+
+LIBRARY_PATH = pv_library_path('')
+
+MODEL_PATH = pv_model_path('')
+
+
+def create(context_path, library_path=None, model_path=None, sensitivity=0.5):
+    """
+    Factory method for Rhino Speech-to-Intent engine.
+
+    :param context_path: Absolute path to file containing context parameters. A context represents the set of
```

Comment:
Suggest explicitly mentioning that this param refers to files with ".rhn" extension since that is how they are usually delivered. Context parameters => context model?